### ISSUE TITLE
[Bug] DoomRunner 1.7.2 concatenates -savedir value and save file name in -loadgame value

### ISSUE THREAD

**Issue by bryanburke** [Youda008/DoomRunner#113]
- DoomRunner version: 1.7.2
- GZDoom version: 4.10.0
- OS: Windows and Linux

When **More options -> Alternative paths -> Save dir** has a value, DoomRunner appends `-savedir <savedir_value>` to the GZDoom launch command as expected.

![image](https://github.com/Youda008/DoomRunner/assets/1231924/9b9daa33-c226-4117-8888-061f826de764)

```sh
... -savedir "./foo" ...
```

However, when **More options -> Launch mode -> Load saved game** is selected and has a value, DoomRunner appends `-loadgame <savedir_value>/<savefile_value>` to the launch command, effectively concatenating the save directory and the save file name.

![image](https://github.com/Youda008/DoomRunner/assets/1231924/d27bbad6-e32c-4b97-99d9-416da1c078be)

```sh
... -loadgame "./foo/save00.zds" -savedir "./foo" ...
```

This causes GZDoom 4.10.0 (I did not test earlier versions) to try to load a save game file at `./foo//./foo/save00.zds` and thus crash with the error message:

> Cannot find savegame ./foo//./foo/save00.zds

I believe the fix is for DoomRunner to provide only the save file name as the `-loadgame` value:

```sh
... -loadgame "save00.zds" -savedir "./foo" ...
```

Note that I did not test older GZDoom versions, so I do not know if this bug is a result of a change in GZDoom behavior.

Thanks for your time!

**Comment by Youda008** [comment]
Thanks for reporting! This should be easy to fix.
I was just about to release a new version with bugfixes yesterday, but luckily for us both i postponed it, so i can include this in it. Laziness saved me once again 😁

**Comment by Youda008** [comment]
Should be fixed in the recently released 1.7.3.
